src/test/software/amazon/event/ruler/ComparableNumberBenchmark.java

```diff
@@ -0,0 +1,78 @@
+package software.amazon.event.ruler;
+
+import com.fasterxml.jackson.core.io.doubleparser.FastDoubleParser;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.infra.Blackhole;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+import org.openjdk.jmh.runner.options.TimeValue;
+
+import java.util.Random;
+
+@State(Scope.Benchmark)
+public class ComparableNumberBenchmark {
```

Comment:
I'm hesitant on merging this one because it would lead to just one benchmark being disconnected from others. Is it possible to push this into a separate branch? we can merge for this one later, when I've brought rest of the benchmark tests to JMH as well.

Reply:
agree, I did backup my changes adding jmh to another branch.